src/backend/launcher.ts

```diff
@@ -1207,6 +1210,9 @@ function setupWineEnvVars(gameSettings: GameSettings, gameId = '0') {
   if (isLinux && !gameSettings.enableFsync && wineVersion.type === 'proton') {
     ret.PROTON_NO_FSYNC = '1'
   }
+  if (isLinux && !gameSettings.enableNtsync && wineVersion.type === 'proton') {
+    ret.PROTON_USE_NTSYNC = '0'
+  }
```

Comment:
You should also handle the case for wine, like in case of `WINEFSYNC` we should have `WINENTSYNC=1`

Reply:
Upstream Wine will have ntsync enabled by default with no variable to control if it's on or off